test/unit/object/test_retention_policy.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import unicode_literals, absolute_import
+
+import json
+
+from boxsdk.config import API
+from boxsdk.object.retention_policy import RetentionPolicy
+from boxsdk.object.retention_policy_assignment import RetentionPolicyAssignment
+
+
+def test_get(test_retention_policy, mock_box_session):
+    expected_url = '{0}/retention_policies/{1}'.format(API.BASE_API_URL, test_retention_policy.object_id)
+    mock_box_session.get.return_value.json.return_value = {
+        'type': 'retention_policy',
+        'id': test_retention_policy.object_id,
+    }
+    retention_policy = test_retention_policy.get()
+    mock_box_session.get.assert_called_once_with(expected_url, headers=None, params=None)
+    assert isinstance(retention_policy, RetentionPolicy)
```

Comment:
You should add more data to the mock API response and assert about it here; this method could be calling the translator incorrectly and not attaching the response object but this test wouldn't catch it